ui/shared/ad/GetitBanner.tsx

```diff
@@ -0,0 +1,48 @@
+import { Flex, chakra } from '@chakra-ui/react';
+import dynamic from 'next/dynamic';
+import React from 'react';
+import { useAccount } from 'wagmi';
+
+import useIsMobile from 'lib/hooks/useIsMobile';
+
+import Web3ModalProvider from '../Web3ModalProvider';
+
+const GetitAdPlugin = dynamic(() => import('getit-sdk').then(module => module.GetitAdPlugin), { ssr: false });
+
+const GETIT_API_KEY = 'ZmGXVvwYUAW4yXL8RzWQHNKmpSyQmt3TDXsXUxqFqXPdoaiSSFyca3BOyunDcWdyOwTkX3UVVQel28qbjoOoWPxYVpPdNzbUNkAHyFyJX7Lk9TVcPDZKTQmwHlSMzO3a';
```

Comment:
shouldn't it be configured via ENV variable?

Reply:
we don't configure api keys for ad providers bc there's no such case when user need a custom api key (yet)